File: assign2.py
# declaring variables of the given matrix
a=int
b=int
#Given matrix
A=[[5,a],[b,0]]
#Declaring transpose of th egiven matrix
TranA=A[:][:]
#Finding Transpose of A
for i in range (2) :
    for j in range (2) :
        TranA[i][j]=A[j][i]
#Verifying whether the matrix is symmetric or not
for i in range (2) :
    for j in range (2) :
        if A[i][j] != TranA[i][j] :
            print("There is an error")
            quit()
print("The matrix A is symmetric")
print("Verified")

File: assign2.c
#include <stdio.h>

int main(){
    int a,b;
    a=b;//for verification
    int A[2][2] = { {5,a} , {b,0} }; //given matrix
    int Atran[2][2];//Transpose of A
    for(int i=0;i<2;i++){
        for(int j=0;j<2;j++){
            Atran[i][j]=A[j][i];
        }
    }
    /*for(int k=0;k<2;k++){
        for(int l=0;l<2;l++){
            printf("%d ",Atran[k][l]);
        }
    }*/
    for(int i=0;i<2;i++){
            for(int j=0;j<2;j++){
            if(A[i][j] != Atran[i][j]){
                printf("There is an error\n");
                return -1;
            }
        }
    }
    printf("A is symmetric\n");
    printf("Verified");
    return 0;
}